wallet-adapter/README.md

```diff
@@ -0,0 +1,46 @@
+# Sui Wallet Adapter
+
+This project is an adapter for wallets on the Sui blockchain.
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```

Comment:
it's non-standard to use headings like this, but it renders OK. Consider using a list